spec/support/shared_context.rb

```diff
@@ -107,7 +107,7 @@ def self.k_with_dsplat_payload(**_args)
         30
       end
       notifies :k_with_ssplat, :k_with_dsplat, :k_with_ssplat_error
-      notifies :k_with_dsplat_payload, { id: 1, first_name: 'Joe', last_name: 'Schmoe' }
+      notifies [:k_with_dsplat_payload, { id: 1, first_name: 'Joe', last_name: 'Schmoe' }]
```

Comment:
Again, hoping this change is not needed.

Reply:
I reverted this back as well as it's not needed - please see earlier [comment](https://github.com/pboling/debug_logging/pull/84#discussion_r516850231)